GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -72,7 +72,8 @@ public override bool IsProcessActive(int processId)
 
         public override void IsServiceInstalledAndRunning(string name, out bool installed, out bool running)
         {
-            throw new NotImplementedException();
+            installed = true;
+            running = true;

```

Comment:
This was another open item.

We check if the service is running so that we can unmount all repositories. If the service is not running, then we will not be able to communicate with the service to unmount repositories.